concent_api/utils/admin.py

```diff
@@ -0,0 +1,28 @@
+
+
+class ReadOnlyMixin:
```

Comment:
I think the name should be more specific. It can't be mixed into anything. Only into admin related stuff. It's in the `admin` module, but still.

Reply:
:+1: done